avni-server-api/src/main/java/org/avni/web/request/api/ApiProgramEncounterRequest.java

```diff
@@ -3,9 +3,11 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public class ApiProgramEncounterRequest extends ApiBaseEncounterRequest {
+
+    @JsonProperty("Program enrolment external Id")
```

Comment:
for consistency....."Enrolment external id" instead of "Program enrolment external id"

Reply:
correct, changed it to Enrolment external ID.